source/Grid/Grid.js

```diff
@@ -381,6 +384,31 @@ export default class Grid extends PureComponent {
     })
   }
 
+  /**
+   * Ensure offset position is visible
+   * Useful for animating position changes
+   */
+  scrollToPosition ({
+    scrollLeft = 0,
+    scrollTop = 0
```

Comment:
Oh that's a really good point. I was imagining you'd always pass in your x and y here but it's perfectly reasonable to only want to move on one access.